docs/supported_formats/renishaw.rst

```diff
@@ -0,0 +1,16 @@
+.. _renishaw-format:
+
+Renishaw
+--------
+
+Reader for spectroscopy data saved using Renishaw's WiRE software.
+Currently, RosettaSciIO can only read the ``.wdf`` format from Renishaw.
+
+If `LumiSpy <https://lumispy.org>`_ is installed, ``Luminescence`` will be
+used as the ``signal_type``.
+
```

Comment:
Mention that development is based on the `py-wdf-reader`, the matlab package and the Gwyddion module also here and not only in the header of the file.

Add note that tested on a limited set of data files.

Reply:
I extended the documentation. Do you have any extra suggestions?